src/test/java/components/HomePageImpl.java

```diff
@@ -25,7 +28,8 @@ public void addItemToCart() {
 
     @Override
     public SummaryPageImpl goToCheckout() {
-        driver.manage().timeouts().implicitlyWait(2500, TimeUnit.MILLISECONDS);
+        WebDriverWait wait = new WebDriverWait(driver,10);
+        wait.until(ExpectedConditions.visibilityOfElementLocated(checkout));
```

Comment:
Nice change, explicit wait instead of implicit! Should make the test run faster now.